docs/proposals/kubelet-eviction.md

```diff
@@ -76,8 +76,10 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
 | nodefs.available   | nodefs.available := node.stats.fs.available |
 | nodefs.inodesFree | nodefs.inodesFree := node.fs.inodesFree |
+| nodefs.inodesPercentAvailable | nodefs.inodesPercentAvailable := node.fs.inodesFree / node.fs.inodes |
```

Comment:
It is already a string. `--eviction-hard` takes in a string. The string can be a series of `key < value` pairs separated by `,`. We will have to parse the `value` as either a percentage (`--eviction-hard=nodefs.inodesFree<10%`), or as an absolute value (`--eviction-hard=nodefs.inodesFree<1000`)